Automattic/wp-calypso
# Fix the Google Docs connection

This PR adds a description, example and Google Drive icon to the Google Drive connection manager setting.
It also fixes the Google Docs connection by handling it as a Calypso service in the same way Google Photos works.
The Google Drive connection is protected by a feature flag "google-drive" in config/development.json

#### Test

* Change to the fix/google_docs_connection branch.
* Go to http://calypso.localhost:3000/marketing/connections and pick a blog
* Note that the Google Drive connection now has an icon and description.
* Connect to your Google Account by clicking the "Connect" button. You may have problems in Firefox if you are using separate containers for Google and your development tab. Make sure they're using the same container profile.
* When connected, the button will show "Disconnect".

Before:
![Screenshot 2022-10-13 at 17 10 45](https://user-images.githubusercontent.com/5656673/195649524-867b2fa0-ee4d-4745-911f-b7c0b780afe7.jpg)

After:
![Screenshot 2022-10-13 at 17 09 52](https://user-images.githubusercontent.com/5656673/195649651-b043b7bd-cff6-429e-b3d7-a25e3600c13b.jpg)

#### Pre-merge Checklist
